locator/network_topology_strategy.cc

```diff
@@ -268,9 +269,9 @@ network_topology_strategy::calculate_natural_endpoints(
 }
 
 void network_topology_strategy::validate_options(const gms::feature_service& fs) const {
-    if(_config_options.empty()) {
-        throw exceptions::configuration_exception("Configuration for at least one datacenter must be present");
-    }
+    // #22688 / #20039 - we want to remove dc:s once rf=0, and we
+    // also want to allow fully setting rf=0 in _all_ dc:s (hello data loss)
```

Comment:
Or a bizarre way to truncate all tables in the keyspace

Reply:
Yup. Still data loss. (Willing)